contracts/RandomGenerator.sol

```diff
@@ -0,0 +1,74 @@
+// SPDX-License-Identifier: MIT
+pragma solidity ^0.6.0;
+
+import "https://github.com/smartcontractkit/chainlink/blob/master/evm-contracts/src/v0.6/VRFConsumerBase.sol";
```

Comment:
use this github URL isn't reliable, import "@chainlink/contracts/src/v0.6/VRFConsumerBase.sol";